apps/emqx_exproto/src/emqx_exproto.appup.src

```diff
@@ -1,22 +1,22 @@
 %% -*- mode: erlang -*-
 {VSN,
-  [{<<"4\\.3\\.[4-5]">>,
+  [{<<"4\\.3\\.[4-5]$">>,
```

Comment:
It's not necessary to add `^` or `$` in appup files, as the systools_relup.erl always matches the complete version string: https://github.com/erlang/otp/blob/9e23eca89170fcdf22441ed65f05ef891096f318/lib/sasl/src/systools_relup.erl#L504

Also see the [docs for appup](https://erlang.org/doc/man/appup.html#file-syntax):

> <<"2\\.1\\.[0-9]+">>
> Notice that the regular expression must match the complete version string, so this example works for, for example, 2.1.1, but not for 2.1.1.1.

Reply:
thanks.
TIL!